token_auth/serializers/register.py

```diff
@@ -0,0 +1,45 @@
+from django.contrib.auth import get_user_model
+from rest_framework import serializers
+from rest_framework_jwt.serializers import PasswordField
+
+User = get_user_model()
+
+class RegisterSerializer(serializers.ModelSerializer):
+
+    password = PasswordField(
+        write_only=True,
+        required=True
+    )
+    password2 = PasswordField(
+        write_only=True,
+    )
+
+    class Meta:
+        model = User
+        fields = [
+            "username",
+            "email",
+            "password",
+            "password2",
+        ]
+
+    def create(self, validated_data):
+
+        username = validated_data["username"]
```

Comment:
always use `.get()` so we don't get runtime exceptions and add validations as we need

Reply:
@Monal5031 You mean instead of create??

Reply:
validated_data is a dict right? So while working with dictionaries always use `.get()` to retrieve a value so you can provide a default as well which can be used for adding checks.
ex:
```py
username = validated_data.get('username', None)
if username is None:
    ...
```